hledger-lib/Hledger/Data/Posting.hs

```diff
@@ -404,8 +404,8 @@ accountNamePostingType a
 
 accountNameWithoutPostingType :: AccountName -> AccountName
 accountNameWithoutPostingType a = case accountNamePostingType a of
-                                    BalancedVirtualPosting -> T.init $ T.tail a
-                                    VirtualPosting -> T.init $ T.tail a
+                                    BalancedVirtualPosting -> maybe "" fst . T.unsnoc $ T.drop 1 a
+                                    VirtualPosting -> maybe "" fst . T.unsnoc $ T.drop 1 a
```

Comment:
Can't this give a different result from before, with a one char account name ? Perhaps not observable in practice, but.. perhaps better to use textUnbracket here ?

Reply:
Seems reasonable.